UPGRADE-4.2.md

```diff
@@ -0,0 +1,7 @@
+UPGRADE FROM 4.1 to 4.2
+=======================
+
+Security
+--------
+
+ * Using the "has_role()" function in security expressions is deprecated, use the "is_granted()' function instead.
```

Comment:
Use backticks instead of quotes?

Reply:
yeah, would be much better.

Or if you really want to use quotes, use the same one on both sides at least.